core/node/options_network.go

```diff
@@ -32,4 +32,7 @@ type OptionsNetwork struct {
 	EtherPaymentsAddress string
 
 	QualityOracle string
+
+	DisableMetrics bool
```

Comment:
It has nothing to do with network options.
Network options are ment to be related to our network itself - blockchain, market etc.

Reply:
Will fix that.